app/services/idv/steps/verify_step.rb

```diff
@@ -24,13 +24,18 @@ def enqueue_job
           document_capture_session,
           should_proof_state_id: should_use_aamva?(pii_from_doc),
           trace_id: amzn_trace_id,
+          document_expired: document_expired,
```

Comment:
High level flow step 3: we take `document_expired` and pass it to `proof_resolution`